tinylog-api-scala/src/main/scala/org/tinylog/scala/Logger.scala

```diff
@@ -22,24 +22,27 @@ import scala.language.experimental.macros
 	*/
 object Logger {
 
-	private val instance = new TaggedLogger(null)
-	private val loggers = new ConcurrentHashMap[String, TaggedLogger]()
+	private val instance = new TaggedLogger(Set(null))
+	private val loggers = new ConcurrentHashMap[Set[String], TaggedLogger]()
 
 	/**
 		* Gets a tagged logger instance. Tags are case-sensitive.
 		*
 		* @param tag
 		* Tag for logger or `null` for receiving an untagged logger
+		* @param otherTags
+		* Additional tags for the logger
 		* @return Logger instance
 		*/
-	def tag(tag: String): TaggedLogger = {
-		if (tag == null || tag.isEmpty()) {
+	def tag(tag: String, otherTags: String*): TaggedLogger = {
```

Comment:
I would prefer two different methods: `tag(tag: String)` and `tags(tags: String*)`